### ISSUE TITLE
Inconsistent drawer menu look on iPad for http://foundation.fsharp.org/*

### ISSUE THREAD

**Issue by TahaHachana** [fsharp/fsharp.org#396]
The drawer menu on the foundation pages is a bit offside on iPads as shown in the attached screenshot. Issue originally spotted by @vasily-kirichenko
![ipaddrawermenu](https://cloud.githubusercontent.com/assets/1418447/5694543/ddf83506-9964-11e4-8795-50927e5ec267.jpg)

**Comment by ReedCopsey** [comment]
Yes, and it's on my list to try to work on.  Unfortunately, as that's using an entirely different framework with their own templating engine, it's going to be a non-trivial task to rework their templates to use the exact same theme.  (The current site there was modified to match fsharp.org *mostly*, but is not anywhere near 100% the same.)

**Comment by dsyme** [comment]
@ReedCopsey @TahaHachana - should we close this or fix it?  Feels like we can live with this inconsistency?

**Comment by TahaHachana** [comment]
Overriding the margin-right property on #topnav might fix this, also setting width to 100% allows the menu to take the entire available space. I'm testing in Chrome's emulator and not a device here, someone with an iPad would better verify this. 
![fsharporgdrawermenufix](https://cloud.githubusercontent.com/assets/1418447/9056219/5fb16052-3a89-11e5-91cd-8bbd08e98a48.PNG)

**Comment by bbatsov** [comment]
Seems to me that's another issue we can close.

**Comment by dsyme** [comment]
Thanks!
